godot-codegen/src/special_cases/special_cases.rs

```diff
@@ -88,9 +88,15 @@ pub fn is_godot_type_deleted(godot_ty: &str) -> bool {
     // See e.g. https://github.com/GodotVR/godot-xr-tools/issues/479.
     // OpenXR is also not available on iOS: https://github.com/godotengine/godot/blob/13ba673c42951fd7cfa6fd8a7f25ede7e9ad92bb/modules/openxr/config.py#L2
     // Do not hardcode a list of OpenXR classes, as more may be added in future Godot versions; instead use prefix.
-    #[cfg(any(all(before_api = "4.2", target_os = "macos"), target_os = "ios"))]
+    let target_os = std::env::var("CARGO_CFG_TARGET_OS").unwrap_or_else(|_| "unknown".to_string());
     if godot_ty.starts_with("OpenXR") {
-        return true;
+        if target_os == "ios" {
+            return true;
+        }
+        if target_os == "macos" {
+            #[cfg(before_api = "4.2")]
+            return true;
+        }
     }
```

Comment:
I think this could be moved inside `if`.

Also, you might want to use `match` against `Some("ios")`, `Some("macos")` and `None` -- then you wouldn't need the "unknown" case.

Reply:
I have updated with match statement